osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -89,6 +92,8 @@ public abstract class DrawableHitObject : SkinReloadableDrawable, IHasAccentColo
 
         public readonly Bindable<ArmedState> State = new Bindable<ArmedState>();
 
+        public readonly BindableBool HasFailed = new BindableBool();
```

Comment:
This shouldn't be here. There's no reason to make DHO more heavy with new variables than it is.

Reply:
then where should it be?, it's used for not applying the fail animation more than once

Reply:
Just allow it to be?